tests/robot-tests/.env.example

```diff
@@ -1,4 +1,5 @@
 PUBLIC_URL=http://localhost:3000  # URL of the public frontend
+PUBLIC_URL_WITH_WWW=http://www.localhost:3000  # URL of the public frontend with www
```

Comment:
We added a `user navigates to public frontend with www` keyword which means we don't need to add a new environment variable.

Can use this keyword for the UI test changes in this PR.

Reply:
D'oh, yes of course we did. Will do